packages/paste-website/src/pages/page-templates/object-details/index.mdx

```diff
@@ -0,0 +1,151 @@
+export const meta = {
+  title: 'Object details',
+  description:
+    'The Object details page template allows customers to view and take action on properties of a single object.',
+  slug: '/page-templates/object-details/',
+};
+
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Card} from '@twilio-paste/card';
+import {UnorderedList, ListItem} from '@twilio-paste/list';
+
+import {SidebarCategoryRoutes} from '../../../constants';
+import DefaultLayout from '../../../layouts/DefaultLayout';
+import {getFeature, getNavigationData} from '../../../utils/api';
+
+export default DefaultLayout;
+
+export const getStaticProps = async () => {
+  const navigationData = await getNavigationData();
+  const feature = await getFeature('Object details');
+  return {
+    props: {
+      data: {
+        ...feature,
+      },
+      navigationData,
+    },
+  };
+};
+
+<content>
+
+<NormalizedPatternHeader
+  description={meta.description}
+  name={meta.title}
+  data={props.data}
+  categoryRoute={SidebarCategoryRoutes.PAGE_TEMPLATES}
+/>
+</content>
+
+---
+
+<contentwrapper>
+
+<PageAside data={mdxHeadings} />
+
+<content>
+
+## About Object details
+
+Object details give customers a primarily read-only view of the properties of an object, such as a message log or sent email.
+
+If the customer is primarily editing information, use the [Settings](/page-templates/settings) page instead.
+
+## Accessibility
+
+A well-structured [document hierarchy](https://www.w3.org/WAI/tutorials/page-structure/) helps provide efficient in-page navigation for keyboard users, assistive technologies, and mobile web users.
+
+Proper hierarchy allows users to skip directly to content that is most relevant to them. This is especially important on an Object details page where there may be many different types of information and customers are more likely to skim through.
+
+To structure a page:
+
+- Use [Headings](/components/heading) to organize the page.
+- Use Headings in sequential order. For example, don’t place an H4 directly after an H2.
+- Use sentence case for headers (“Message date” not “Message Date”).
+- Use a [Description List](/components/description-list) to define relationships between a property and its label.
+- Use [Tables](/components/table) to organize data that are meant to be compared.
+
+## Ingredients
+
+<Box display="flex" flexDirection="row" columnGap="space130" justifyContent="center" marginBottom="space70">
```

Comment:
suggestion: I'm increasingly more convinced that components created in mdx files are pretty dangerous as they aren't typechecked or linted. Creating it in a tsx component file elsewhere and importing it, might be a better way going forward

Reply:
done

still using anchor and list components in the mdx file tho